janusgraph-server/src/main/java/org/janusgraph/graphdb/grpc/schema/SchemaManagerImpl.java

```diff
@@ -73,7 +59,11 @@ public void getVertexLabelByName(
         GetVertexLabelByNameRequest request,
         StreamObserver<GetVertexLabelByNameResponse> responseObserver
     ) {
-        SchemaManagerProvider provider = getSchemaManagerProvider(request, responseObserver::onError);
+        if (request == null) {
+            responseObserver.onError(getRequestIsRequiredError());
+            return ;
+        }
+        SchemaManagerProvider provider = getSchemaManagerProvider(request::getContext, responseObserver::onError);
```

Comment:
I am curious why not pass the value of `request.getContext()`?

Reply:
Good idea